demoScripts/aksQuickStart.md

```diff
@@ -0,0 +1,426 @@
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: azure-vote-back
+    spec:
+      ports:
+      - port: 6379
+      selector:
+        app: azure-vote-back
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: azure-vote-front
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: azure-vote-front
+      template:
+        metadata:
+          labels:
+            app: azure-vote-front
+        spec:
+          nodeSelector:
+            "kubernetes.io/os": linux
+          containers:
+          - name: azure-vote-front
+            image: mcr.microsoft.com/azuredocs/azure-vote-front:v1
+            resources:
+              requests:
+                cpu: 100m
+                memory: 128Mi
+              limits:
+                cpu: 250m
+                memory: 256Mi
+            ports:
+            - containerPort: 80
+            env:
+            - name: REDIS
+              value: "azure-vote-back"
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: azure-vote-front
+    spec:
+      type: LoadBalancer
+      ports:
+      - port: 80
+      selector:
+        app: azure-vote-front
+
+    EOF
+    ```
+      For a breakdown of YAML manifest files, see [Deployments and YAML manifests](../concepts-clusters-workloads.md#deployments-and-yaml-manifests).
+
+1. Deploy the application using the [kubectl apply][kubectl-apply] command and specify the name of your YAML manifest:
+
+    ```azurecli-interactive
+    kubectl apply -f azure-vote.yaml
+    ```
+
+    The following example resembles output showing the successfully created deployments and services:
+    <!--expected_similarity=0.8-->
+    ```output
+    deployment "azure-vote-back" created
+    service "azure-vote-back" created
+    deployment "azure-vote-front" created
+    service "azure-vote-front" created
+    ```
+
+## Test the application
+
+When the application runs, a Kubernetes service exposes the application front-end to the internet. This process can take a few minutes to complete.
+
+Monitor progress using the [kubectl get service][kubectl-get] command.
+
+```azurecli-interactive
+timeout 30 kubectl get service azure-vote-front --watch
+```
+
+The **EXTERNAL-IP** output for the `azure-vote-front` service will initially show as *pending*.
+<!--expected_similarity=0.4-->
+```output
+NAME               TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+azure-vote-front   LoadBalancer   10.0.37.27   <pending>     80:30572/TCP   6s
+```
+
+Once the **EXTERNAL-IP** address changes from *pending* to an actual public IP address, use `CTRL-C` to stop the `kubectl` watch process. The following example output shows a valid public IP address assigned to the service:
+
+```output
+azure-vote-front   LoadBalancer   10.0.37.27   52.179.23.131   80:30572/TCP   2m
+```
+
+To see the Azure Vote app in action, open a web browser to the external IP address of your service.
```

Comment:
We need to test this otherwise there is no guarantee that the application was deployed correctly. 

A downside of this approach is the `timeout` approach is that we would need a separate command to get the IP number for testing. I do, however, prefer the readability of this approach, especially if are further able to simplify with the "special command" approach.

What we could do is something like:

It is often useful to have the IP address in an environment variable for later reuse. You can do this by running:

```
export QUICKSTART_AKS_IP=...
```

Reply:
Totally agree - that is a good catch. I thought I validated the output but that must have been on a separate tutorial.